zerver/models.py

```diff
@@ -230,7 +230,12 @@ class Meta(object):
 post_save.connect(flush_realm, sender=Realm)
 
 class RealmAlias(models.Model):
-    realm = models.ForeignKey(Realm, null=True) # type: Optional[Realm]
+    class Meta():
+        # same as adding db_index=True below, but realm wasn't indexed
+        # originally and so we need to do this for makemigrations to see it
+        index_together = [['realm']]
```

Comment:
I think this whole hunk is not required; ForeignKey comes with an index automatically.